tests/test_http_app/test_app.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import patch
+
+import pytest
+from fastapi.testclient import TestClient
+
+from imaginairy.http_app.app import app
+
+client = TestClient(app)
+
+
+@pytest.mark.asyncio()
+async def test_imagine_endpoint():
```

Comment:
great. will look cleaner with `monkeypatch` pytest fixture

Reply:
since this is reused. also just make it into a test fixture

Reply:
done